samples/bluetooth/central_hids/src/main.c

```diff
@@ -131,6 +138,11 @@ static void disconnected(struct bt_conn *conn, u8_t reason)
 
 	printk("Disconnected: %s (reason %u)\n", addr, reason);
 
+	if (bt_gatt_hids_c_assign_check(&hids_c)) {
+		printk("HIDS client active - releasing");
```

Comment:
Please think about switching to LOG. `hids` uses logger already. But this can be done in next PR.

Reply:
Ok, I got it, there's printk in the sample code and log in library.

Reply:
Exactly.